src/Parameters/InsertDocumentParameters.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of littleredbutton/bigbluebutton-api-php.
+ *
+ * littleredbutton/bigbluebutton-api-php is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * littleredbutton/bigbluebutton-api-php is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with littleredbutton/bigbluebutton-api-php. If not, see <http://www.gnu.org/licenses/>.
+ */
+namespace BigBlueButton\Parameters;
+
+/**
+ * @method string getMeetingID()
+ * @method $this setMeetingID(string $id)
+ */
+class InsertDocumentParameters extends MetaParameters
```

Comment:
```suggestion
final class InsertDocumentParameters extends MetaParameters
```

Personally, I would make classes `final` if we don't consider them as an extension point. There should be no reason to be able to extend the class.

Reply:
Property would be also switched to `protected` then. But I am also ok with that, when we keep consistency for now and not switch to `final`. But I would reconsider that for a major release.